terraform/projects/app-ecs-services/config/alerts/alerts.yml

```diff
@@ -1,14 +1,28 @@
 groups:
-- name: AlertManager
+- name: PrometheusCore
   rules:
   - alert: AlertManager_Below_Threshold
     expr: up{job="alertmanager"} == 0 and on(job) sum by(job) (up{job="alertmanager"}) <= 1
     for: 10s
     labels:
         severity: "P2"
-        product: "tools"
+        product: "prometheus"
     annotations:
         summary: "Service is below the expected instance Threshold"
         description: "The service name is {{ $labels.job }}. The URL experiencing the issue is {{ $labels.instance }}"
+  - alert: NoFileSdTargets
+    # this expression is a little weird - count() has no value instead of 0 if there are no
+    # matching metrics.  But no value isn't less than 1 so we aren't able to trigger an alert
+    # that a regular count() is zero.  So we force missing values to be equal to
+    # zero using the `or count(up)*0` expression.
+    # This idea taken from: https://www.robustperception.io/existential-issues-with-metrics/
+    expr: (count(prometheus_sd_file_mtime_seconds) or count(up)*0) < 1
+    for: 10s
+    labels:
+        severity: "P2"
```

Comment:
Do we have to add a severity here? Happy to if we do just a slight worry about doing enacting this without a shared understanding of severity levels

Reply:
although it's just occurred to me that we're using P2 in other places too

Reply:
yeah i think it was just here as an example of what labels might look like.  But we have product too, so i think we can kill severity until we define it better 🚮